crates/engine/schema/src/generated/resolver.rs

```diff
@@ -23,44 +23,43 @@ use walker::{Iter, Walk};
 ///   @meta(module: "resolver")
 ///   @variants(
 ///     empty: ["Introspection"]
-///     names: [
-///       "GraphqlRootField"
-///       "GraphqlFederationEntity"
-///       "FieldResolverExtension"
-///       "SelectionSetResolverExtension"
-///       "Lookup"
-///     ]
+///     names: ["GraphqlRootField", "GraphqlFederationEntity", "FieldResolverExtension", "Extension", "Lookup"]
 ///   )
 ///   @indexed(deduplicated: true, id_size: "u32") =
 ///   | GraphqlRootFieldResolverDefinition
 ///   | GraphqlFederationEntityResolverDefinition
 ///   | FieldResolverExtensionDefinition
-///   | SelectionSetResolverExtensionDefinition
+///   | ExtensionResolverDefinition
 ///   | LookupResolverDefinition
 /// ```
 #[derive(serde::Serialize, serde::Deserialize)]
 pub enum ResolverDefinitionRecord {
+    Extension(ExtensionResolverDefinitionRecord),
```

Comment:
Could this be a breaking change? Remember to test the postgres extension with a new CLI if it builds with that before merging.

Reply:
nothing breaking yet and yes still works